CHANGELOG.md

```diff
@@ -22,6 +22,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
```

Comment:
Haha sorry, I should have said this in my previous comment... 🤦‍♀️ - Could you put this under ### Samples section instead?